plugin/src/main/groovy/gorm/tools/repository/MangoRepoEntity.groovy

```diff
@@ -0,0 +1,11 @@
+package gorm.tools.repository
+
+import groovy.transform.CompileStatic
+import javax.persistence.Transient
+
+@CompileStatic
+trait MangoRepoEntity {
+
+    @Transient
+    static List<String> quickSearchFields = []
```

Comment:
@alexeyzvegintcev  static fields are really recomended on Traits as they act funny. does this work across domains? I don't see any tests.